sonar-project.properties

```diff
@@ -0,0 +1,13 @@
+sonar.projectKey=open-cluster-management_go-template-utils
+sonar.projectName=go-template-utils
+sonar.organization=open-cluster-management
+sonar.sources=.
+sonar.exclusions=**/*_test.go,**/*_generated*.go,**/*_generated/**,**/vendor/**,/test/**,/build/**,/vbh/**,/version/**
+sonar.tests=.
+sonar.test.inclusions=**/*_test.go
+sonar.test.exclusions=**/*_generated*.go,**/*_generated/**,**/vendor/**,**/test/e2e/**
+sonar.go.tests.reportPaths=report.json,report_e2e.json,report_unit.json
+sonar.go.coverage.reportPaths=coverage.out,coverage_e2e.out,coverage_unit.out
```

Comment:
I don't think the tests produce a coverage report.

Reply:
They do in the `sonar/js/prow` target in `build-harness-extensions`: https://github.com/stolostron/build-harness-extensions/blob/ff4a318ef269064f2752cdf8bdf1ec0105172bde/modules/sonar/Makefile#L30